lms/djangoapps/branding/api.py

```diff
@@ -170,6 +170,47 @@ def _footer_social_links():
     return links
 
 
+def _build_support_form_url(full_path=False):
+    """
+    Return the support form path
+
+    Returns url of support form, which can have 3 possible values:
+    - '' if the contact form is disabled (by setting SiteConfiguration
+       CONTACT_US_PAGE = True), by the wich is ignored by the
+       parent function
+    - The normal edx support form path if CONTACT_US_PAGE is not set
+    - CONTACT_US_PAGE if the the user has set a custom URL redirect
+      for support forms
+
+    Parameters:
+        - full_path: bool. Appends base_url to returned value if
+                     CONTACT_US_PAGE = True
+
+    Returns: string
+
+    """
+    # The contact us page is enabled by default
+    contact_us_conf = configuration_helpers.get_value('CONTACT_US_PAGE', True)
+
+    if contact_us_conf:
+        # If contact_us_conf is bool enable contact form if it's string,
+        # set link to url
+        if isinstance(contact_us_conf, bool):
+            contact_us_page = reverse("support:contact_us")
+
+            if full_path:
+                return '{base_url}{contact_us_path}'.format(
```

Comment:
Isn't one of the use cases meant to enable entirely external contact forms (e.g. `https://help.edx.org`)? If so, we shouldn't be prepending the LMS base URL, right?

Is it possible to just always echo the URL out as it was entered in configuration, so they could do whatever combination of absolute/relative/fully-qualified link they needed?

Reply:
If we are using external contact forms this piece of code isn't run because of  `if isinstance(contact_us_conf, bool):` a few lines above.

This was implemented because one link on the footer used full path and the other one used relative path and I wanted to keep the current behaviour and not break any tests.